GRAPHICS/GetScaleformMovieFunctionReturnString.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: GRAPHICS
-aliases: ["SITTING_TV"]
+aliases: ["SITTING_TV", "_GET_SCALEFORM_MOVIE_FUNCTION_RETURN_STRING"]
 ---
-## _GET_SCALEFORM_MOVIE_FUNCTION_RETURN_STRING
+## GET_SCALEFORM_MOVIE_FUNCTION_RETURN_VALUE_STRING
```

Comment:
Same here, this native name does not match the native joaat hash. Please prefix the name with `_`.

(`GET_SCALEFORM_MOVIE_FUNCTION_RETURN_VALUE_STRING` = `0xFE8D588C`, the Joaat hash of this native is: `0x516862EB`)

Also the file name should be changed when changing the native name.